features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/SearchFragment.kt

```diff
@@ -400,7 +400,8 @@ class SearchFragment : Fragment(R.layout.fragment_search) {
 
     private fun transitToDetails() {
         val item = viewModel.selectedItem.value ?: return
-        val isOnline = item.type == MerchantType.ONLINE
+        val isOnline = viewModel.filterMode.value == FilterMode.Online ||
+                item.type == MerchantType.ONLINE
```

Comment:
When the item is of Both type, we treat it like Online if it is opened from the Online tab.